packages/mediacenter/kodi/package.mk

```diff
@@ -12,6 +12,12 @@ PKG_DEPENDS_TARGET="toolchain JsonSchemaBuilder:host TexturePacker:host Python3
 PKG_LONGDESC="A free and open source cross-platform media player."
 PKG_BUILD_FLAGS="+speed"
 
+post_unpack() {
+  # don't build internal TexturePacker
+  sed -i 's|set(INTERNAL_TEXTUREPACKER_INSTALLABLE TRUE)|set(INTERNAL_TEXTUREPACKER_INSTALLABLE FALSE)|' \
+    $(get_build_dir kodi)/cmake/modules/FindTexturePacker.cmake
+}
+
```

Comment:
IMO it'd be better to handle this via a patch.
eg kodi-100.26-disable-internal-texturepacker.patch
```
--- a/cmake/modules/FindTexturePacker.cmake
+++ b/cmake/modules/FindTexturePacker.cmake
@@ -52,10 +52,8 @@ if(NOT TARGET TexturePacker::TexturePacker::Executable)
       endif()
     endif()
 
-    # Ship TexturePacker only on Linux and FreeBSD
-    if(CMAKE_SYSTEM_NAME STREQUAL "FreeBSD" OR CMAKE_SYSTEM_NAME STREQUAL "Linux")
-      set(INTERNAL_TEXTUREPACKER_INSTALLABLE TRUE)
-    endif()
+    # Don't install TexturePacker
+    set(INTERNAL_TEXTUREPACKER_INSTALLABLE FALSE)
 
     # Use it during build if build architecture is same as host
     # (not cross-compiling) and TEXTUREPACKER_EXECUTABLE is not found
```

Reply:
No need to set INTERNAL_TEXTUREPACKER_INSTALLABLE to FALSE.